### PR TITLE
Sex is required

### PR DESCRIPTION
**Pull request by captDaylight**
Sorry, not sorry, for crazy PR title. But this is my task ref https://www.notion.so/photons/Sex-is-not-required-on-frontend-when-it-is-required-for-the-API-15f46c740b254b93badcace18a18c33c?pvs=4

Found other fields that were not being validated on patient create form including first name, last name, phone, and email.

### CODE REVIEW

**Comment by captDaylight** [comment]
Are we enforcing addresses on new patients? right now it’s using the logic from update patient which only makes you fill in more address information if one of the address fields is touched… but I imagine we’ll want to capture that information on patient creation

File: packages/elements/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@photonhealth/elements",
-  "version": "0.0.81",
+  "version": "0.0.82",
```
Comment (samkotlove) [inline packages/elements/package.json:3]: So is this already published?

File: packages/elements/src/photon-patient-dialog/index.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ customElement(
       const event = new CustomEvent('photon-patient-closed', {
         composed: true,
         bubbles: true,
-        detail: {},
+        detail: {}
```
Comment (samkotlove) [inline packages/elements/src/photon-patient-dialog/index.tsx:50]: do we have a linting file in here?

File: packages/elements/src/photon-patient-form/index.tsx [hunk 3]
```diff
@@ -58,30 +59,34 @@ customElement(
...
-      validator: message(size(string(), 1, Infinity), 'Please enter a last name...'),
+      validator: message(size(string(), 1, Infinity), 'Please enter a last name...')
+    });
+    actions.registerValidator({
+      key: 'sex',
+      validator: message(enums(sexes.map((s) => s.name)), 'Please enter Sex at Birth...')
     });
     actions.registerValidator({
       key: 'phone',
-      validator: message(size(string(), 10, 12), 'Please enter a phone number...'),
```
Comment (samkotlove) [inline packages/elements/src/photon-patient-form/index.tsx:73]: Isn't 10 a minimum?

File: packages/elements/src/photon-patient-dialog/index.tsx (hunk 2 above)
Comment (captDaylight) [reply packages/elements/src/photon-patient-dialog/index.tsx:50]: we do, it's global to the repo now, was gonna run  it on a separate PR so it doesn't blow up the commits

File: packages/elements/src/photon-patient-form/index.tsx (hunk 3 above)
Comment (captDaylight) [reply packages/elements/src/photon-patient-form/index.tsx:73]: it is, but the input converts 2223334444 (10) into +12223334444 (12)

File: packages/elements/package.json (hunk 1 above)
Comment (captDaylight) [reply packages/elements/package.json:3]: nope, I'll publish it when it lands on boson

**Review by samkotlove — APPROVED** [review]

**Pull request merged**